content/en/docs/Configuration/p8s-jaeger-grafana/tracing/tempo.md

```diff
@@ -40,13 +47,26 @@ spec:
   external_services:
     grafana:
       in_cluster_url: http://grafana.istio-system:3000
-      url: http://my-grafana-host
+      url: https://grafana.apps-crc.testing/
 ```
 
-We also need to set up a default [Tempo datasource](https://grafana.com/docs/grafana/latest/datasources/tempo/) in Grafana. 
+#### Set up a Tempo Datasource in Grafana
+
+We also need to set up a default [Tempo datasource](https://grafana.com/docs/grafana/latest/datasources/tempo/) in Grafana: 
```

Comment:
This actually is optional, right?  This is only needed in order to see the trace data in the Grafana UI and to fully complete the Kiali integration, as I understand it. So I would just mention this as being "optional" because it is not necessarily required in order for Kiali to work with  Tempo and for Kiali UI itself to show the Tempo traces.

And this screen shot does not show setting it up - this actually shows what Grafana UI looks like AFTER the datasource is set up and you are looking at trace data. So a slight re-wording should be done for that, too, to make it clear what this next screenshot is actually showing.

```suggestion
We can optionally set up a default [Tempo datasource](https://grafana.com/docs/grafana/latest/datasources/tempo/) in Grafana so that you can view the Tempo tracing data within the Grafana UI, as you see here: 
```